client/src/modules/cash/modals/invoice-modal.html

```diff
@@ -30,7 +30,7 @@
   </div>
 
   <div ng-show="CashInvoiceModalCtrl.empty" class="text-warning">
-    <i class="fa fa-info-circle"></i> {{ ::"FORM.WARNINGS.EMPTY_SELECTION" | translate }}
+    <i class="fa fa-info-circle"></i> <span translate> {{ ::"FORM.WARNINGS.EMPTY_SELECTION" }} </span>
```

Comment:
You can remove the `{{ :: }}`.  The two colons `::` was trying to make the binding more efficient - they are angular's one-time binding.  But since we are moving to the directive, no need for binding at all.